### DISCUSSION TITLE
assignments using codespaces

### DISCUSSION THREAD

**Discussion by denoc1** [community/community#138066] [GitHub Education]
### Hi everyone,

Could someone help a newbie to the community?  I'm new to GitHub, GitHub Codespaces and GitHub Classroom. 
The issue I am having is that when I create a new assignment, codespaces is not showing up as an option as an online editor.  Makecode, VS Code and "Don't use online IDE" are the only options. 
Troubleshooting:
 I have been verified as a teacher (at least there is a box saying that on the teacher verification page) however, in my classroom settings, GitHub Codespaces section says I still need to be verified.  Could this be why I don't see codespaces as an option?
I was sure to check, in my organization settings, "Enable for all members and outside collaborators:
Enable GitHub Codespaces for all organization members on all organization owned private and internal repositories"
Everything else is working - I just can't get it so students to have access to codespaces through the assignments.  
I would appreciate help.  I have spent hours trying to figure this out!  Thank you!

**Comment by VadymMakohon** [comment]
It sounds like the issue is related to the teacher verification process for GitHub Codespaces. Even though you're verified as a teacher, your Classroom settings indicate that Codespaces still requires verification. This mismatch is likely preventing Codespaces from appearing as an option.

Steps to resolve:
Ensure full teacher verification for Codespaces: Visit [GitHub Education](https://education.github.com/) to check if the verification for Codespaces specifically is completed.
Recheck organization settings: Ensure that Codespaces is enabled correctly in your classroom's organization settings under "Codespaces".
Contact GitHub Support: If the issue persists, contact [GitHub Support](https://support.github.com/contact) to assist with the Codespaces verification.
This should help resolve the issue with Codespaces not showing up as an editor option for assignments.
